googletest/test/gtest-port_test.cc

```diff
@@ -75,8 +75,8 @@ TEST(IsXDigitTest, WorksForNarrowAscii) {
 }
 
 TEST(IsXDigitTest, ReturnsFalseForNarrowNonAscii) {
-  EXPECT_FALSE(IsXDigit(static_cast<char>(0x80)));
-  EXPECT_FALSE(IsXDigit(static_cast<char>('0' | 0x80)));
+  EXPECT_FALSE(IsXDigit(static_cast<char>(0x80u)));
+  EXPECT_FALSE(IsXDigit(static_cast<char>('0' | 0x80u)));
```

Comment:
This is not more concise but perhaps it expresses the intent better:

  static_cast<char>(std::numeric_limits<signed char>::min())

This is the most concise:

  '\x80'

although for the one where '0' is ORed in, the static_cast<char>() is still needed around the entire expression.